### PR TITLE
Edit delete on workspace

### PR DESCRIPTION
**Pull request by KojoSwiss**
I added the edit and delete to the booking and its view.

### CODE REVIEW

File: app/views/shared/_form.html.erb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <div class="container mg-tp-md">
   <div class="row justify-content-center">
     <div class="col-lg-6">
-      <h1>Create a workspace</h1>
```
Comment (Simon-Busch) [inline app/views/shared/_form.html.erb:4]: Why did you delete it ?

File: app/views/workspaces/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Edit Workspace</h1>
+<%= render 'shared/form'%>
```
Comment (Simon-Busch) [inline app/views/workspaces/edit.html.erb]: Is it working ?

**Review by Simon-Busch — CHANGES_REQUESTED** [review]
Please review my comments

**Review by ericblandet — CHANGES_REQUESTED** [review]
should not the form stay as a partial in workspace, and not in shared?

File: app/views/shared/_form.html.erb (hunk 1 above)
Comment (ericblandet) [reply app/views/shared/_form.html.erb:4]: looks like it is in the form already

**Review by ericblandet — APPROVED** [review]
